mappings/net/minecraft/client/gui/screen/ingame/BeaconScreen.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_783 net/minecraft/client/gui/screen/ingame/BeaconScreen
+	FIELD field_3408 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_3409 beaconTexture Lnet/minecraft/class_1605;
```

Comment:
did mojang concern and make this not static final

Reply:
I'll look into that